### PR TITLE
docs: add rider configuration documentation

### PR DESCRIPTION
**Pull request by mozhoku**
Adding documentation for Jet Brains Rider installation & configuration to Unity Editor.

### CODE REVIEW

**Comment by xmfcx** [comment]
You should also update the .mkdocs.yaml page like this commit: https://github.com/autowarefoundation/AWSIM/commit/dde4c6cd4edc57fd7768be838ea428a92446cb14

File: docs/DeveloperGuide/EditorSetup/JetBrains Rider/index.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+## Connect Rider to Unity Editor:
+
+- https://www.jetbrains.com/help/rider/Unity.html#8f092cb7_9
+
+```bash
+1) Open an existing Unity project in the Unity Editor.
+
+2) Select Edit | Preferences (Unity | Settings on macOS) and open the External Tools page.
+
+3) In the External Script Editor, select a $p1//product-profile[@id=$p2]/@name installation.
```
Comment (xmfcx) [inline docs/DeveloperGuide/EditorSetup/JetBrains Rider/index.md]:
```suggestion
3) In the External Script Editor, select a $p1//product-profile[@id=$p2]/@name installation.
```
What is `$p1//product-profile[@id=$p2]/@name`?

**Comment by mozhoku** [comment]
Updated mkdocs.yaml file as pointed out by @xmfcx. :+1:

**Comment by mozhoku** [comment]
Updated rider doc @xmfcx .

**Review by xmfcx — APPROVED** [review]
Thx!

**Pull request merged**